lexer_schemas/commerce_api/transaction_event.py

```diff
@@ -68,7 +68,8 @@ class PurchaseEvent(BaseTransactionEvent):
     purchase_id: str = Field(examples=["53059a..."])
     payment_types: Optional[List[PaymentType]] = None
     products: List[PurchaseProductReference] = Field(
-        description="Contains the 'order lines' in the purchase event. Each record in this array represents a specific product that was purchased including details such as quantity, price paid etc."
+        description="Contains the 'order lines' in the purchase event. Each record in this array represents a specific product that was purchased including details such as quantity, price paid etc.",
```

Comment:
I don't think we want to impose this restriction - it's useful to allow empty orders to be loaded, e.g. for cancelled orders or other edge cases.

The learn article should just be updated to remove reference the minimum size restriction.